netty-http authentication tests fail due to NPE in native mode

### Bug description

Both `testCredentials` & `testAcls` are failing. It needs more investigation why the NPE occurs.

```
Caused by: java.lang.NullPointerException
	at org.apache.camel.quarkus.component.http.netty.NettyHttpResource.auth(NettyHttpResource.java:189)
	at org.apache.camel.quarkus.component.http.netty.NettyHttpResource_ClientProxy.auth(Unknown Source)
```

--- a/extensions/netty-http/deployment/src/main/java/org/apache/camel/quarkus/component/netty/http/deployment/NettyHTTPProcessor.java
+++ b/extensions/netty-http/deployment/src/main/java/org/apache/camel/quarkus/component/netty/http/deployment/NettyHTTPProcessor.java
@@ -18,6 +18,7 @@ package org.apache.camel.quarkus.component.netty.http.deployment;
 
 import io.quarkus.deployment.annotations.BuildStep;
 import io.quarkus.deployment.builditem.FeatureBuildItem;
+import io.quarkus.deployment.builditem.nativeimage.NativeImageResourceBundleBuildItem;
 import io.quarkus.deployment.builditem.nativeimage.ReflectiveClassBuildItem;
 
 class NettyHTTPProcessor {
@@ -43,5 +44,10 @@ class NettyHTTPProcessor {
                         "com.sun.security.auth.module.NTLoginModule",
                         "com.sun.jmx.remote.security.FileLoginModule")
                 .build();
+    }
+
+    @BuildStep
+    NativeImageResourceBundleBuildItem nativeImageResoourceResourceBundles() {
+        return new NativeImageResourceBundleBuildItem("sun.security.util.resources.security");
     }
 }

--- a/integration-test-groups/http/netty-http/src/test/java/org/apache/camel/quarkus/component/http/netty/it/NettyHttpTest.java
+++ b/integration-test-groups/http/netty-http/src/test/java/org/apache/camel/quarkus/component/http/netty/it/NettyHttpTest.java
@@ -29,7 +29,6 @@ import org.apache.camel.quarkus.component.http.common.AbstractHttpTest;
 import org.apache.camel.quarkus.component.http.common.HttpTestResource;
 import org.apache.camel.quarkus.test.support.certificate.TestCertificates;
 import org.eclipse.microprofile.config.ConfigProvider;
-import org.junit.jupiter.api.Disabled;
 import org.junit.jupiter.api.Test;
 import org.junit.jupiter.params.ParameterizedTest;
 import org.junit.jupiter.params.provider.CsvSource;
@@ -144,7 +143,6 @@ public class NettyHttpTest extends AbstractHttpTest {
                 .body(is("proxy"));
     }
 
-    @Disabled("https://github.com/apache/camel-quarkus/issues/8147")
     @ParameterizedTest
     @CsvSource({
             "null,null,401",
@@ -162,7 +160,6 @@ public class NettyHttpTest extends AbstractHttpTest {
                 .statusCode(responseCode);
     }
 
-    @Disabled("https://github.com/apache/camel-quarkus/issues/8147")
     @ParameterizedTest
     @CsvSource({
             "admin,admin,adminpass,200",

